web/mattermost-ssl

```diff
@@ -1,34 +0,0 @@
-# server {
```

Comment:
Yes, I have no strong opinion with that.
I have [many projects](https://github.com/indiehosters/) that I maintain, and I usually don't ship an ssl option (because I wouldn't maintain it).
That's mainly why I deleted it. But feel free to keep it. I understand you point of view.